tests/unittests/tensorsTest.cpp

```diff
@@ -652,6 +652,43 @@ TEST(Tensor, insertWithCountAndAxis) {
   }
 }
 
+TEST(Tensor, zeroQuanitizedTensor) {
+  Tensor Q8T(ElemKind::Int8QTy, {3, 4, 5, 6}, 127, 0);
+  Tensor Q16T(ElemKind::Int16QTy, {3, 4, 5}, 1234.7, 223);
+  Tensor Q32T(ElemKind::Int32QTy, {3, 4}, 500.4, 53452);
+
+  auto Q8H = Q8T.getHandle<int8_t>();
+  EXPECT_TRUE(Q8H.isZero());
+  for (size_t i = 0; i < 3 * 4 * 5 * 6; i++) {
```

Comment:
Here and other loops below:

```suggestion
  for (size_t i = 0, e = Q8H.size(); i < e; i++) {
```